_observability-plugin/trace/trace-analytics-jaeger.md

```diff
@@ -0,0 +1,238 @@
+    ports:
+      - "5775:5775/udp"
+      - "6831:6831/udp"
+      - "6832:6832/udp"
+      - "5778:5778"
+    networks:
+      - opensearch-net
+    restart: on-failure
+    environment:
+      - SPAN_STORAGE_TYPE=opensearch
+    depends_on:
+      - jaeger-collector
+
+  hotrod:
+    image: jaegertracing/example-hotrod:latest
+    ports: 
+      - "8080:8080"
+    command: ["all"]
+    environment:
+      - JAEGER_AGENT_HOST=jaeger-agent
+      - JAEGER_AGENT_PORT=6831
+    networks:
+      - opensearch-net
+    depends_on:
+      - jaeger-agent
+
+volumes:
+  opensearch-data1:
+  opensearch-data2:
+
+networks:
+  opensearch-net:
+```
+
+### Step 2: Start the cluster
+
+Run the following command to deploy the Docker compose YAML file.
+
+```
+docker compose up -d
+```
+To stop the cluster, run the following command:
+
+``` 
+docker compose down
+```
+
+## Step 2: View trace data in OpenSearch Dashboards
+
+After you generate Jaeger trace data you can go to OpenSearch Dashboards to view your trace data.
+
+Go to OpenSearch Dashboards Trace Analytics at [Trace Analytics](http://localhost:5601/app/observability-dashboards#/trace_analytics/home).
+
+
+## Use trace analytics in OpenSearch Dashboards
+
+To analyze your Jaeger trace data in the Dashboards, you need to set up Trace Analytics first. To get started, see [Get started with Trace Analytics]({{site.url}}{{site.baseurl}}/observability-plugin/trace/get-started/).
+
+### Data sources
+
+You can specify either Data Prepper or Jaeger as your data source when you perform trace analytics.
+From the OpenSearch Dashboards, go to **Observability > Trace Analytics** and select Jaeger.
+
+![Select data source]({{site.url}}{{site.baseurl}}/images/trace-analytics/select-data.png)
+
+## Dashboards views
+
+After you select Jaeger for the data source, you can view all of your indexed data in **Dashboard** view including **Error rate** and **Throughput**.
+
+### Error rate
+
+You can view the Trace error count over time in the Dashboard, and also view the top five combinations of services and operations that have a non-zero error rate.
+
+![Error rate]({{site.url}}{{site.baseurl}}/images/trace-analytics/error-rate.png)
+
+### Throughput
+
+With **Throughput** selected, you can see the throughput of traces on Jaeger indexes that are coming in over time.
+
+You can select an individual Trace from **Top 5 Service and Operation Latency** list and view the detailed trace data.
+
+![Throughput]({{site.url}}{{site.baseurl}}/images/trace-analytics/throughput.png)
+
+You can also see the combinations of services and operations that have the highest latency.
+
+If you select one of the entries for Service and Operation Name and go to the **Traces** column to select a trace, it will add the service and operation as filters for you.
+
+In **Traces**, you can see the latency and errors for the filtered service and operation for each individual Trace ID in the list.
```

Comment:
Traces is also a separate sub page, do we want to give it the same treatment as ###services?

Reply:
ok thanks